events.html

```diff
@@ -1,17 +1,19 @@
 ---
 layout: default
-title: Events
+title: Online and In-Person Events
 ---
 
-<div class="page-header">
-    <h1>Online and In-Person Events</h1>
-</div>
-<div class="container">
-    <div class="content-constrained">
-        <div class="section">
-            <p>Women Coding Community events and meetups present great opportunities to improve your skill set, deepen your understanding of various topics, expand your professional network, and engage in meaningful social interactions.</p>
-            <p>Our community hosts various engaging events, including webinars, hands-on technical workshops, study groups, panel discussions and keynotes featuring industrial experts and tech professionals. Whether online or in-person, these gatherings provide an invaluable opportunity to share both professional insights and personal experiences with fellow members. Through collaborative learning and networking, participants can broaden their horizons, stay abreast of the latest trends, and exchange knowledge within the community.</p>
-            <p>Join us on Slack in <a href="https://womencodingcommunity.slack.com/archives/C0700RUUTJR" target="_blank">#Events</a> channel to stay up to date with the upcoming events.</p>
+<div class="page events">
+    {% include page-header.html %}
```

Comment:
now I am wondering if would be better to had added this as part of the layout file, instead been adding in all pages 🙈